doc/source/changes/version_0_26.rst.inc

```diff
@@ -10,6 +10,11 @@
 * renamed `na` argument of `read_csv`, `read_excel`, `read_hdf` and `read_sas` functions to `fill_value` to avoid
   confusion as to what the argument does and to be consistent with `reindex` and `align` (closes :issue:`394`).
 
+* renamed several methods (closes :issue:`50`):
```

Comment:
I think we should explain the motivation for any syntax change. eg. 
* to improve consistency with other methods by always using "index" instead of using a mix of "position" (P) and "index" (i).
* to make the meaning of the "arg" methods (hopefully) more intuitive.

You also need to mention PGroup -> IGroup